### PR TITLE
docs(control-api): add some notes for healthcheck API

### PR DESCRIPTION
**Pull request by soulbird**
### Description

Add some notes for healthcheck API. 

fix https://github.com/apache/apisix/issues/7141

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: docs/zh/latest/control-api.md [hunk 1]
```diff
@@ -195,6 +195,8 @@ APISIX 中一些插件添加了自己的 control API。如果你对他们感兴
 }
 ```
 
+注意：API 请求命中的进程如果没有触发健康检查，那么本次 API 请求将不会获取到任何健康检查数据。
```
Comment (hf400159) [inline docs/zh/latest/control-api.md:198]:
```suggestion
:::note

API 请求命中的进程如果没有触发健康检查，那么本次 API 请求将不会获取到任何健康检查数据。

:::
```

File: docs/en/latest/control-api.md [hunk 2]
```diff
@@ -198,6 +198,8 @@ For example, `GET /v1/healthcheck/upstreams/1` returns:
 }
 ```
 
+Note: If the process hit by the API request does not trigger the health check, the API request will not obtain any health check data.
```
Comment (hf400159) [inline docs/en/latest/control-api.md:201]:
```suggestion
:::note

If the process hit by the API request does not trigger the health check, the API request will not obtain any health check data.

:::
```

File: docs/zh/latest/control-api.md (hunk 1 above)
Comment (hf400159) [reply docs/zh/latest/control-api.md:198]: please refer: [admonitions](https://docusaurus.io/docs/next/markdown-features/admonitions)

**Review by hf400159 — DISMISSED** [review]
LGTM

File: docs/en/latest/control-api.md [hunk 3]
```diff
@@ -198,6 +198,12 @@ For example, `GET /v1/healthcheck/upstreams/1` returns:
 }
 ```
 
+:::note
+
+If the process hit by the API request does not trigger the health check, the API request will not obtain any health check data.
```
Comment (spacewander) [inline docs/en/latest/control-api.md:203]:
```suggestion
As APISIX uses multiple-process architecture,  the health check information of an upstream might not appear until it is hit by a request handled in the current process.
```

**Review by spacewander — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Pull request merged**